vmdb/app/models/storage.rb

```diff
@@ -100,6 +100,12 @@ def my_zone
     return self.ext_management_systems.first.my_zone
   end
 
+  def self.without_hosts
+    habtm_table = Arel::Table.new(:hosts_storages)
+    join_table_with_condition = habtm_table.project(habtm_table[:storage_id])
```

Comment:
We have a model for HostStorage, so no need to build a query like this, I think.